### PR TITLE
Use the new AbstractBrowser class from BrowserKit

### PR DESCRIPTION
**Pull request by dunglas**
Required to prevent deprecations.

### CODE REVIEW

**Comment by dunglas** [comment]
fabpot failure are false positives

File: composer.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     ],
     "require": {
         "php": ">=5.5.0",
```
Comment (dunglas) [inline composer.json]: Actually, we could also bump the PHP version, and other Symfony dependencies.

Comment (fabpot) [reply composer.json]: Please, let's do that. No need to still maintain such an old PHP version.

Comment (adrienlucas) [reply composer.json]: Assuming this would be done in a separate MR, I propose #387  
Feel free to cherry-pick or whatever :)

**Comment by fabpot** [comment]
Thank you @dunglas.

**Pull request merged**